src/lio/bin/lio_mv.c

```diff
@@ -194,13 +197,15 @@ int main(int argc, char **argv)
         } else {
             info_printf(lio_ifd, 0, "ERROR: Multiple paths selected but the dest(%s) isn't a directory!\n", dtuple.path);
         }
+        n_errors = 1;
```

Comment:
It's really more of a binary value.  0=Success and anything else means a 
failure.  You could change the final return to:
         return((n_errors == 0) ? 0 : EIO)